styles/properties.less

```diff
@@ -520,18 +524,51 @@ label.bpp-hidden {
   opacity: 0.7;
 }
 
+.bpp-collapsible-error .bpp-collapsible__description:not(:empty) {
```

Comment:
Maybe we can standardize to not have selectors for any type of collapsible item. But that's for sure not part of this PR.

Reply:
Yeah I wasn't too sure about that, would you see an immediate change to apply?

Reply:
not immediately no, let's keep it in our mind

also cc @barmac

Reply:
Not sure what we can do about it right now. I hope we move someday out of the current collapsible trick.